src/substrait/textplan/converter/tests/BinaryToTextPlanConversionTest.cpp

```diff
@@ -120,6 +119,73 @@ std::vector<TestCase> GetTestCases() {
                        "  function multiply:opt_fp64_fp64 as multiply;\n"
                        "}\n"),
       },
+      {
+          "read local files",
+          R"(relations: {
+            root: {
+              input: {
+                read: {
+                  local_files {
+                    items {
+                      partition_index: 0
+                      start: 0
+                      length: 3719
+                      uri_file: "/mock_lineitem.orc"
+                      orc {}
+                    }
+                  }
+                }
+              }
+            }
+          })",
+          AllOf(
+              HasSymbols({"local", "read", "root"}),
+              SerializesTo(
+                  "read relation read {\n"
```

Comment:
Why not an `R"()"` string here?  Is that because of the line endings?

Reply:
I was worried about whitespace (I wasn't planning on indenting) but now that you mention it the line endings are going to be a problem.  I'll look at upgrading the matchers sooner than later so we can start using raw literals in the output.

Reply:
Done.